src/articleView-components/Article.jsx

```diff
@@ -0,0 +1,20 @@
+import { useState } from "react";
+export function Article({ article }) {
+  // const [articleBody, setArticleBody]=useState('')
+
+  return (
+    <>
+      <div className="solo-article-container">
+        <h1> {article.title}</h1>
+        <h2>by {article.author}</h2>
+        <img className="solo-article-img" src={article.article_img_url}></img>
+        <article className="solo-article-body">{article.body}</article>
+        <p className="solo-article-votes">Votes: {article.votes}</p>
+      </div>
+      <form>
```

Comment:
Not sure this needs to be a form, when you get to it just handlers on the buttons will be grand.